lsp/nls/src/server.rs

```diff
@@ -59,6 +59,7 @@ impl Server {
             references_provider: Some(OneOf::Left(true)),
             completion_provider: Some(CompletionOptions {
                 trigger_characters: Some(vec![String::from(DOT_COMPL_TRIGGER)]),
+                resolve_provider: Some(true),
```

Comment:
Ok, the culprit was this line. We don't really provide a resolver I think, so if that's fine by you, let's remove it.
```suggestion
```